manifests/test_templates.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test-templates
```

```suggestion
  generateName: test-templates
```

prodvana will fail when they try apply this if we dont generate a different name for each run